gradle.properties

```diff
@@ -45,8 +45,10 @@ org.gradle.parallel=true
 
 # kotlin
 kotlin.native.ignoreDisabledTargets=true
+kotlin.native.enableKlibsCrossCompilation=true
```

Comment:
In theory, it should make it simpler for contributors to test their changes

Reply:
please make sure publication works as it is

Reply:
It seems to break our [allocation tests](https://ktor.teamcity.com/buildConfiguration/Ktor_AllocationTests/309678). So I disable this flag for now.